runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightConfig.java

```diff
@@ -52,4 +52,15 @@ default boolean chatboxTooltip()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			position = 2,
+			keyName = "spellbookTooltip",
+			name = "Disable Spellbook Tooltips",
+			description = "Disable Spellbook Tooltips so they don't cover descriptions"
```

Comment:
Reduce indents by one level.
```suggestion
		position = 2,
		keyName = "spellbookTooltip",
		name = "Disable Spellbook Tooltips",
		description = "Disable Spellbook Tooltips so they don't cover descriptions"
```